core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java

```diff
@@ -75,6 +81,31 @@ public AntennaWorkflow(AntennaWorkflowConfiguration antennaWFConfig) {
         }
     }
 
+    private void logFailCausingResults(Map<String, Set<IEvaluationResult>> failCausingResults) {
+        LOGGER.error(makeStringForFailCausingResults(failCausingResults));
+    }
+
+    private String makeStringForFailCausingResults(Map<String, Set<IEvaluationResult>> failCausingResults) {
+        StringBuilder failCausingResultMessage = new StringBuilder("Build fails due to fail causing results \n");
+
+        failCausingResults.forEach((workflowItemName, evaluationResults) ->
+                failCausingResultMessage.append(workflowItemName)
+                        .append(" failed with evaluation results: \n")
+                        .append(makeStringForEvaluationResults(evaluationResults))
+                        .append("\n"));
+
+        return failCausingResultMessage.toString();
+    }
+
+    private String makeStringForEvaluationResults(Set<IEvaluationResult> evaluationResults) {
+        StringBuilder evaluationResultsString = new StringBuilder();
+        evaluationResults.forEach(evaluationResult ->
+                evaluationResultsString.append(evaluationResult.getId())
+                .append(" failed with: ")
+                .append(evaluationResult.getDescription()));
+        return evaluationResultsString.toString();
```

Comment:
I miss the artifact that failed here as information, helps a lot in finding the issue :-)